scripts/build_jswrapper.py

```diff
@@ -313,10 +353,16 @@ def codeOutBuiltins(indent, builtin):
 codeOut('');
 codeOut('');
 
-codeOut('const JswSymList jswSymbolTables[] = {');
+# output the strings, possibly with __attribute__ to put them into flash
```

Comment:
You can make this conditional on the -B{board-type} command arg:

````
if "-BESP8266_BOARD" in sys.argv:
    codeOut("static const JswSymPtr jswSymbols_"+codeName+"[] FLASH_SECT= {\n  "+",\n  ".join(listSymbols)+"\n};");
  else:
    codeOut("static const JswSymPtr jswSymbols_"+codeName+"[] = {\n  "+",\n  ".join(listSymbols)+"\n};");
````

This way the generated code does not have to have as many @ifdef conditionals